src/components/ResultView.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ResultView = ({ obj }) => (!obj ? (
+  <div className="result">
+    <p>0</p>
+  </div>
+) : (
+  <div className="result">
+
+    <p>
+      {obj.total}
+      {obj.operation}
+      {obj.next}
+    </p>
+  </div>
+));
+
+export default ResultView;
+
+ResultView.defaultProps = {
+  obj: 0,
+};
+ResultView.propTypes = {
+  obj: PropTypes.number,
+};
```

Comment:
![image](https://github.com/Mahamadou-Nouridine/math-magicians/assets/46839548/894aa715-2574-47c3-bd73-062d134f6d66)

- [ ] An error occurs in the browser console because of the invalid prop supplied to it. I suggest that you use `PropTypes.shape` to further describe the properties of the object you are expecting 😎 See hint below 👇 
```javascript
ResultView.propTypes = {
  obj: PropTypes.shape({
    total: PropTypes.string,
    operation: PropTypes.string,
    next: PropTypes.string,
  }),
};
```

Reply:
Thank you for the tip